CHANGELOG.md

```diff
@@ -85,6 +85,7 @@
 * [5777](https://github.com/grafana/loki/pull/5777) **tatchiuleung**: storage: make Azure blobID chunk delimiter configurable
 * [5650](https://github.com/grafana/loki/pull/5650) **cyriltovena**: Remove more chunkstore and schema version below v9
 * [5643](https://github.com/grafana/loki/pull/5643) **simonswine**: Introduce a ChunkRef type as part of logproto
+* [6349](https://github.com/grafana/loki/pull/6349) **simonswine**: Updates default HTTP listen port from 80 to 3100, make sure to configure the port explicitly if you are using port 80.
```

Comment:
Thanks for adding this.
Can you please add a section in `docs/sources/upgrading/_index.md` about this? This could cause some confusion in the upgrade path